src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,133 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    @Override
+    public Book create(Book book) {
+        try {
```

Comment:
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/20326858/642b0f3e-591b-440c-ac3d-c455770a5c73)

Reply:
not fixed